### PR TITLE
[docs] Add expo accounts with personal and members info

### PR DESCRIPTION
**Pull request by byCedric**
# Why

We are adding some functionality like programmatic access to Expo, and we need some info for that 👍 

# How

- Added `Expo Accounts` as a group in sidebar under "guides"
- Added `Personal Account` info with programmatic access under this group
- Added `Working together` info with members and roles under this group

# Test Plan

Docs only

### CODE REVIEW

**Comment by byCedric** [comment]
@tcdavis we also have this page, https://docs.expo.io/guides/account-permissions/. I think we should remove this and redirect users to "Working Together", right?

File: docs/pages/accounts/working-together.md
Comment (tcdavis) [inline docs/pages/accounts/working-together.md:19]: The owner permission here and on line 15 should be moved to the Orgs docs changes.  Owner only applies to Organization accounts

File: docs/pages/accounts/working-together.md
Comment (tcdavis) [inline docs/pages/accounts/working-together.md:5]: Until Orgs lands collaboration features are only available to Dev Services users.  I'd replace this line with something like "You can grant other users access to the projects belonging to your Personal Account with Expo Teams.  The type of access depends on the granted role.  You can sign up for Teams on any of your accounts at https://expo.io/dashboard/ACCOUNT/settings/services"

**Review by tcdavis — APPROVED** [review]
Yeah, totally agree with dropping https://docs.expo.io/guides/account-permissions/ and redirecting to the working together page.  Looks great outside of a couple changes that should be folded in to the Orgs release
Thanks!

File: docs/pages/accounts/working-together.md
Comment (byCedric) [reply docs/pages/accounts/working-together.md:19]: Good point, I'll strip it from this PR!

File: docs/pages/accounts/working-together.md
Comment (byCedric) [reply docs/pages/accounts/working-together.md]: Good point, I replaced it with the team reference! 👍

**Comment by byCedric** [comment]
Alright, @tcdavis feel free to merge this once we deploy the tokens (if I'm offline).

**Pull request merged**
